pkg/sql/catalog/descpb/structured.proto

```diff
@@ -1219,7 +1219,12 @@ message TableDescriptor {
   // stored outside the span of the object.
   optional ExternalRowData external = 61 [(gogoproto.nullable) = true];
 
-  // Next ID: 62
+  // Replicated ID tracks the original version from the source tenant
+  // that this descriptor was created from.
+  optional uint32 replicated_version = 63 [(gogoproto.nullable) = false,
```

Comment:
nit: I wonder if the name of this field should be `ReplicatedPCRVersion`, which makes it more clear that this field is very specific to PCR.

Also the name in the comment is out of date.